pkg/bugzilla/client.go

```diff
@@ -0,0 +1,82 @@
+package bugzilla
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// Client is implemented by anything with an ExecuteQuery method
```

Comment:
no need to do another revision of this PR, but in the future `is implemented by anything with an ExecuteQuery method` is redundant and not useful info -- you can just say `Client knows how to execute save Bugzilla queries` or something similar..

Reply:
Heard.  I'll tweak this in a future PR